packages/vscode-extension/src/commands/SetupAccount.ts

```diff
@@ -47,28 +47,28 @@ export default class SetupAccount extends BaseAction {
 		super(COMMAND_NAME);
 	}
 
-	protected async execute() {
+	protected async execute(): Promise<void> {
 		const accountsPromise = AuthenticationUtils.getAuthIds(getSdkPath());
 		this.messageService.showStatusBarMessage(this.translationService.getMessage(MANAGE_ACCOUNTS.LOADING), true, accountsPromise);
-		const actionResult: ActionResult<AuthListData> = await accountsPromise;
-
-		if (actionResult.status === actionResultStatus.SUCCESS) {
-			const selected = await this.getAuthListOption(actionResult.data);
-			if (!selected) {
-				return;
-			} else if (selected.option === UiOption.new_authid) {
-				this.handleNewAuth(actionResult.data);
-			} else if (selected.option === UiOption.select_authid) {
-				this.handleSelectedAuth(selected.authId);
+			const actionResult: ActionResult<AuthListData> = await accountsPromise;
+			if (actionResult.isSuccess()) {
+				const selected = await this.getAuthListOption(actionResult.data);
+				if (!selected) {
+					return;
+				} else if (selected.option === UiOption.new_authid) {
+					await this.handleNewAuth(actionResult.data);
+				} else if (selected.option === UiOption.select_authid) {
+					this.handleSelectedAuth(selected.authId);
+				}
+			} else {
+				this.messageService.showCommandError(actionResult.errorMessages[0]);
+				this.vsConsoleLogger.error(actionResult.errorMessages[0]);
```

Comment:
Maybe we just want to `this.messageService.showErrorMessage(actionResult.errorMessages[0]);`  ?
Because in these 2 lines we are populating commandError message and then consoleLogger as well with the same info. 
Or do we want to intentionally leave the message in the SuiteCloud Output window. I'm fine with it, I just want to know if it's done on propose.

Reply:
You are right, now it´s consistent with the other commands when such error occurs. I removed the message from the function and swapped the order

Reply:
Great. Thanks